packages/react/src/search/utils/getSearchRedirectUrl.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  buildQueryStringFromObject,
```

Comment:
It seems like we are missing tests for this file :) 
I believe we can move the outdated tests from the `useSearchIntents` hook related to this util to the new tests file.